src/urdb.h

```diff
@@ -128,8 +128,6 @@ class OpenEI
 	bool QueryUtilityCompanies(wxArrayString &names, wxString *err=NULL);
 	// search by zip code per email from Jay Huggins 1/9/15
 	bool QueryUtilityCompaniesbyZipcode(const wxString &zipcode, wxArrayString &names, wxString *err=NULL);
-	// resolve aliases in database per email from Jay Huggins 1/9/15
-	bool ResolveUtilityName(const wxString &name, wxString *urdb_name, wxString *err=NULL);
```

Comment:
@cpaulgilman, is this no longer needed to resolve things like XCEL to CO Public Service, etc.? Does the search bring up the correct utilities? This was originally used to resolve multiple names for the same utility to a master utility name that actually contained all the rates. The other names were just aliases.

Reply:
@sjanzou Utility name aliases work without the `ResolveUtilityName()` function. Not sure if the API changed since 2015 or if I am missing something. I added the utility name to the URDB Download Window so you can see the full utility company name after choosing a rate to verify the correct utility:

![image](https://user-images.githubusercontent.com/27711933/105640647-fd3e6580-5e33-11eb-8451-ae1986a061ff.png)

Reply:
Great, the other troublesome utility was PG&E (Pacific Gas and Electric, Pacific Gas, Pacific Light) - are these resolved correctly?
I think after that and the painful UI merging - everything looks good to me!

Reply:
Here are ones that I tried that work: PG&E, PGE, Xcel. Pacific Gas and Electric, Pacific Gas, Pacific Light do not work (just typing "Pacific" gives a list of utility names that might match). The functionality in SAM is consistent with the URDB website "Browse rates by utility name" at https://openei.org/wiki/Utility_Rate_Database.